SOURCES.toml

```
@@ -0,0 +1,686 @@
+[[resources]] # Path: 7zip.png
+path        = "7zip.png"
```

I'm not a fan of aligned =. 

```suggestion
path = "7zip.png"
```

is simpler and easier to edit

---

If there is an automatic tool (ruff?) I am okay either way btw. I just don't want to manually align stuff.

---

> If there is an automatic tool (ruff?) I am okay either way btw. I just don't want to manually align stuff.

@domoritz 

- https://taplo.tamasfe.dev/cli/usage/formatting.html
- https://taplo.tamasfe.dev/configuration/formatter-options.html

---

@domoritz
I should've explained this better in the description for (https://github.com/vega/vega-datasets/pull/643/commits/1ea28125fc3e0a0e3973edef04f92866b4b27ed6)

That commit uses [automatic formatting](https://taplo.tamasfe.dev/configuration/formatter-options.html) via [`taplo`](https://taplo.tamasfe.dev/).
The two non-default configuration options applied were:

`align_entries=true`
> Align entries vertically. Entries that have table headers, comments, or blank lines between them are not aligned.

`allowed_blank_lines=1`
> The maximum amount of consecutive blank lines allowed.

---

`taplo` can be used via its [CLI](https://taplo.tamasfe.dev/cli/introduction.html) and as a VSCode extension ([Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml))

I haven't used it in CI before, but it is [available on `PyPI`](https://pypi.org/project/taplo) - so we could run it with `uvx`?
https://github.com/tamasfe/taplo/blob/a8bc571ee28775e7d5ad84c2ea87cf5b61ab42f5/.github/workflows/releases.yaml#L503-L504

[NPM](https://taplo.tamasfe.dev/cli/installation/npm.html) is another option for install

---

If we have a formatter, it should run on the ci as well.

---

@domoritz I've opened #646 for this